docs/swarm/admin_guide.md

```diff
@@ -0,0 +1,243 @@
+worker nodes.
+
+## Operating manager nodes in a swarm
+
+Swarm manager nodes use the [Raft Consensus Algorithm](raft.md) to manage the
+cluster state. You only need to understand some general concepts of the raft
+implementation in order to manage a swarm.
+
+There is no limit to scale for manager nodes. The decision about how many
+manager nodes to implement is a trade-off between performance and
+fault-tolerance. Adding manager nodes to a swarm makes the swarm more
+fault-tolerant. However, additional manager nodes reduce write performance
+because more nodes must acknowledge proposals to update the cluster state.
+This means more network round-trip traffic.
+
+Raft requires a majority of members, also called a quorum of `(N/2)+1`, to
+agree on proposed updates to the cluster. A quorum must also agree on node
+additions and removals. Membership operations are subject to the same
+constraints as state replication.
+
+## Add manager nodes for fault tolerance
+
+You should maintain an odd number of managers in the swarm to support manager
+node failures. Having an odd number of nodes ensures that during a network
+partition, a quorum remains available to process requests.
+
+| Cluster Size |  Majority  |  Fault Tolerance  |
+|:------------:|:----------:|:-----------------:|
+|      1       |     1      |         0         |
+|      2       |     2      |         0         |
+|    **3**     |     2      |       **1**       |
+|      4       |     3      |         2         |
+|    **5**     |     3      |       **2**       |
+|      6       |     4      |         2         |
+|    **7**     |     4      |       **3**       |
+|      8       |     5      |         3         |
+|    **9**     |     5      |       **4**       |
+
+
+For example, in a swarm with *5 nodes*, if you lose *3 nodes*, you don't have a
+quorum. Therefore you can't add or remove nodes until you recover one of the
+unavailable manager nodes or recover the cluster with disaster recovery
+commands. See [Recover from disaster](#recover-from-disaster).
+
+While it is possible to scale a swarm down to a single manager node, it is
+impossible to demote the last manager node. This ensures you maintain access to
+the swarm and that the swarm can still process requests. Scaling down to a
+single manager is an unsafe operation and is not recommended. If
+the last node leaves the cluster unexpetedly during the demote operation, the
+cluster swarm will become unavailable until you reboot the node or restart with
+`--force-new-cluster`.
+
+You manage cluster membership with the `docker swarm` and `docker node`
+subsystems. Refer to [Add nodes to a swarm](join-nodes.md) for more information
+on how to add worker nodes and promote a worker node to be a manager.
+
+## Distribute manager nodes across datacenter regions
+
+In addition to maintaining an odd number of manager nodes, Docker recommends
+that you carefully place managers in the datacenter topology. For optimal
```

Comment:
How about "In addition to maintaining an odd number of manager nodes, pay attention to datacenter topology when placing managers"?